templates/member/index.html.twig

```diff
@@ -17,23 +17,23 @@
 		<div class="container overflow-hidden">
 			<div class="row ">
```

Comment:
ul and li tags

Reply:
It's really necessary? Because in the loop it's a card for each member, or I've missed something